interpreter/runtimecost_test.go

```diff
@@ -306,7 +306,7 @@ func TestRuntimeCost(t *testing.T) {
 			name:  "select: field test only",
```

Comment:
Please also add test case for behavior that regressed to prevent future regressions.

Reply:
Gotcha, I've added an additional test - fixed